app/services/search_payload.rb

```diff
@@ -0,0 +1,36 @@
+class SearchPayload
+  attr_reader :registerable_calendar
```

Comment:
I didn't want to change the model name and thought it would be better to be explicit about what this is. There is a Calendar model in the app, but it's incompatible with this class.